support-frontend/app/controllers/PayPalOneOff.scala

```diff
@@ -102,11 +102,21 @@ class PayPalOneOff(
         "payerId" -> PayerID,
       )
 
+      val similarProductsConsentCookie = request.cookies.get("gu_similar_products_consent")
+
+      val similarProductsConsent = similarProductsConsentCookie.flatMap { cookie =>
+        cookie.value.toLowerCase match {
+          case "true" => Some(true)
+          case "false" => Some(false)
+          case _ => None
+        }
```

Comment:
```suggestion
        cookie.value.toBooleanOption
```
see https://scastie.scala-lang.org/ehFqkaqyTNGZSMlKcKjLxA